src/XrdCrypto/XrdCryptosslX509.cc

```diff
@@ -507,8 +507,8 @@ const char *XrdCryptosslX509::IssuerHash(int alg)
       if (issueroldhash.length() <= 0) {
          // Make sure we have a certificate
          if (cert) {
-            char chash[15] = {0};
-            snprintf(chash,15,"%08lx.0",X509_NAME_hash_old(cert->cert_info->issuer));
+            char chash[30] = {0};
+            snprintf(chash, sizeof(chash), "%08lx.0", X509_NAME_hash_old(cert->cert_info->issuer));
```

Comment:
Hi, the '.0' is because this information is mostly used to load the CA files which by convention are '<hash>.0' (for a reason which is likely to be the one that Brian explained). I agree that this could have been done differently.